### DISCUSSION TITLE
What is the purpose of `.next/cache/images` ?

### DISCUSSION THREAD

**Discussion by andrewfritz86** [vercel/next.js#41893] [Help]
Noticing in my production logs that there is some sort of permissions issue within the Docker container running my app server.

```
Failed to write image to cache PY9tgUAhCTyDdkrEpTzhsauK9s5XcaPxkzlKpyCEZTM= [Error: EACCES: permission denied, mkdir '/usr/src/app/.next/cache/images'] {
```

This can theoretically be solved with some permissions fixes in the Docker file, but my question is: what is this directory for? What might be the consequences of it not existing in this case?

Thanks.

**Comment by jnoe2233** [comment]
I'm running in to the same issue. Changing the write permissions helps but sometimes after this error my app process is not running anymore. Did you find the answer to this?

**Reply by andrewfritz86** [reply]
TLDR - stop using the custom Image component unless you are hosting your app on Vercel. Else, the app will attempt to cache images on the Node server itself and "bounce" calls to this directory, completely defeating the point of a performant image CDN, which any production level site would already be using to host images.
